server/belga/io/feed_parsers/belga_ats_newsml_1_2.py

```diff
@@ -39,10 +39,24 @@ def parse_newsmanagement(self, item, manage_el):
         super().parse_newsmanagement(item, manage_el)
         item['firstcreated'] = item['firstcreated'].astimezone(pytz.utc)
         item['versioncreated'] = item['versioncreated'].astimezone(pytz.utc)
+
+        if item.get('pubstatus') == 'embargoed':
+            item['pubstatus'] = 'usable'
+
         # Source is ATS
         credit = {"name": 'ATS', "qcode": 'ATS', "scheme": "sources"}
         item.setdefault('subject', []).append(credit)
 
+    def parse_contentitem(self, item, content_el):
+        super().parse_contentitem(item, content_el)
+
+        if content_el is None:
+            return
+
+        element = content_el.find('Comment')
```

Comment:
are they sending 1 comment max?

Reply:
Yes, In the ticket few XML files are there and all contained only one comment.